library/xwiki

```diff
@@ -25,15 +25,31 @@ Directory: 12/postgres-tomcat
 GitCommit: 3f09b1d7e286b4c2ff611742c417a118111f5e84
```

Comment:
Doh! it looks like `12.10.1` tagged images actually contain `12.10.2` (https://github.com/xwiki/xwiki-docker/commit/3f09b1d7e286b4c2ff611742c417a118111f5e84)

Reply:
Outch you're right apparently https://github.com/docker-library/official-images/pull/9353 was really wrong we forgot to update the tags for 12.10... I'll provide a new PR for fixing it.